og_ui/og_ui.routing.yml

```diff
@@ -15,3 +15,29 @@ og_ui.settings:
     _title: 'OG settings'
   requirements:
     _permission: 'administer group'
+
+og_ui.roles_permissions_overview:
+  path: 'admin/config/group/{type}'
+  defaults:
+    _controller: '\Drupal\og_ui\Controller\OgUiController::rolesPermissionsOverviewPage'
+    _title_callback: '\Drupal\og_ui\Controller\OgUiController::rolesPermissionsOverviewTitleCallback'
+    type: 'roles'
+  requirements:
+    _permission: 'administer group'
+    type: '^(roles|permissions)$'
+
+og_ui.roles_form:
+  path: 'admin/config/group/roles/{entity_type}/{bundle}'
+  defaults:
+    _form: '\Drupal\og_ui\Form\OgRolesForm'
+    _title: '@todo - create title callback'
```

Comment:
Isn't this the `rolesPermissionsOverviewTitleCallback` method you have in this PR?